packages/desktop-client/src/components/modals/SimpleFinInitialise.tsx

```diff
@@ -0,0 +1,90 @@
+// @ts-strict-ignore
+import React, { useState } from 'react';
+
+import { send } from 'loot-core/src/platform/client/fetch';
+
+import { Error } from '../alerts';
+import { ButtonWithLoading } from '../common/Button';
+import { ExternalLink } from '../common/ExternalLink';
+import { Input } from '../common/Input';
+import { Modal, ModalButtons } from '../common/Modal';
+import type { ModalProps } from '../common/Modal';
+import { Text } from '../common/Text';
+import { View } from '../common/View';
+import { FormField, FormLabel } from '../forms';
+
+type SimpleFinInitialiseProps = {
+  modalProps?: Partial<ModalProps>;
+  onSuccess: () => void;
+};
+
+export const SimpleFinInitialise = ({
+  modalProps,
+  onSuccess,
+}: SimpleFinInitialiseProps) => {
+  const [token, setToken] = useState('');
+  const [isValid, setIsValid] = useState(true);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const onSubmit = async () => {
+    if (!token) {
+      setIsValid(false);
+      return;
+    }
+
+    setIsLoading(true);
+
+    await Promise.all([
```

Comment:
I don't think we need the Promise.all here but other than that LGTM! Thank you! :)